tests/Magento_Page_Loading.php

```diff
@@ -0,0 +1,473 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+class MagentoPageLoading extends Sauce\Sausage\WebDriverTestCase
+{
+    protected $base_url = 'http://127.0.0.1/';
+
+    public static $browsers = [
+        [
+            'browserName' => 'chrome',
+            'desiredCapabilities' => [
+                'version' => '45.0',
+                'platform' => 'OS X 10.10',
+            ],
+        ],
+    ];
+
+    const ADMIN_USERNAME    = 'denis';
+    const ADMIN_PASSWORD    = 'zavy123';
+    const ORDERS_COUNT      = 50;
+
+    const PAGE_TYPE_ADMIN       = 'admin';
+    const PAGE_TYPE_ORDER       = 'order';
+    const PAGE_TYPE_FRONT       = 'front';
+    const PAGE_TYPE_CUSTOMER    = 'customer';
+
+    private static $frontFailList       = [];
+    private static $customerFailList    = [];
+    private static $adminFailList       = [];
+    private static $orderFailList       = [];
+
+    /**
+     * Pages that don't use magento layout
+     *
+     * @var array
+     */
+    private $customLayoutPages = [
+        [
+            'link' => '/doneright/enhanced/dashboard',
```

Comment:
moreover we should move into separate config file all specific data and parameters which can be used by all of us (config.local.php) and by travis CI service (config.staging.php)

domain name => ''
admin domain name => ''
preferred browsers => []
admin_account => [login => 'test', pass=> 'test']
customer_account => [login => 'test', pass=> 'test']
etc